art/attacks/evasion/brendel_bethge.py

```diff
@@ -2055,6 +2055,7 @@ def logits_difference(y_true, y_pred):
                 nb_classes=estimator.nb_classes,
                 input_shape=estimator.input_shape,
                 loss_object=self._loss_object,
+                optimizer=estimator._optimizer,
```

Comment:
Should we use the new property:
```suggestion
                optimizer=estimator._optimizer,
```